spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/chat/client/ChatClientBuilderProperties.java

```diff
@@ -36,12 +36,48 @@ public class ChatClientBuilderProperties {
 	 */
 	private boolean enabled = true;
 
+	private Observation observation = new Observation();
+
+	public Observation getObservation() {
+		return this.observation;
+	}
+
 	public boolean isEnabled() {
-		return enabled;
+		return this.enabled;
 	}
 
 	public void setEnabled(boolean enabled) {
 		this.enabled = enabled;
 	}
 
+	public static class Observation {
+
+		/**
+		 * Whether to include the input content in the observations.
+		 */
+		private boolean includeInput = false;
+
+		/**
+		 * Whether to include the output content in the observations.
+		 */
+		private boolean includeOutput = false;
```

Comment:
I wonder if we need this. The output is already observed from the low-level model, including the completion content.

Reply:
I will remove this along the ChatResponse context field.